packages/contentstack-export/src/export/modules/marketplace-apps.ts

```diff
@@ -47,7 +49,7 @@ export default class ExportMarketplaceApps {
       this.marketplaceAppConfig.dirName,
     );
     await fsUtil.makeDirectory(this.marketplaceAppPath);
-    this.developerHubBaseUrl = this.exportConfig.developerHubBaseUrl || (await getDeveloperHubUrl(this.exportConfig));
+    this.developerHubBaseUrl = this.exportConfig.developerHubBaseUrl || this.command.developerHubUrl;
```

Comment:
Creating a command instance to access the developerhub url is not correct way here, we can get it from the parent class

Reply:
yes got it